services/github/github-labels.service.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { GithubAuthV3Service } = require('./github-auth-service')
+const { documentation, errorMessagesFor } = require('./github-helpers')
+
+const schema = Joi.object({
+  color: Joi.string(),
```

Comment:
IIRC, all GitHub labels must have a color value (the system automatically assigns one if the user does not specify) so I think we can make the schema a bit more restrictive. Let me know if I've got those constraints wrong though 👍 

```suggestion
  color: Joi.string().hex().required(),
```

Reply:
Makes total sense, including the hex.
I didn't know how to use Joi properly.

Reply:
No worries! Looks like one of the inline suggestions I gave was not properly formatted for prettier, so if you could run an `npm run prettier` and then push that should resolve the failing prettier checks